community-operators/aqua/aqua-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,452 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: aqua-operator.v0.0.1
+  namespace: placeholder
+  annotations:
+    capabilities: Basic Install
+    categories: Security
+    description: The Aqua Security Operator runs within a Openshift cluster and provides a means to deploy and manage Aqua Security cluster and components.
+    certified: "false"
+    containerImage: nissimaqua/aqua-operator:0.0.1
+    createdAt: 2019-03-21T08:00:00Z
+    support: Aqua Security, Inc.
+    repository: https://github.com/aquasecurity/aqua-operator
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "operator.aquasec.com/v1alpha1",
+          "kind": "AquaCsp",
+          "metadata": {
+            "name": "aqua"
+          },
+          "spec": {
+            "requirements": true,
+            "password": "password",
+            "rbac": {
+              "enable": true,
+              "openshift": false,
```

Comment:
Please set `"openshift"` to true for the community-operators CSV. Likewise, set it false in `upstream-community-operators`

Reply:
Updated!
Thanks!

Reply:
@SamiSousa Updated